scripts/host-check

```diff
@@ -1255,6 +1305,35 @@ def perform_checks(checks: List[Check]) -> bool:
     return not any(c.is_error() for c in results.values())
 
 
+def run_extra_checks(extra_checks: List[str]) -> Tuple[Set[str], Set[str]]:
+    extras_supported = set()
+    extras_not_supported = set()
+
+    print("")  # insert a newline
+    print_heading("Extra checks")
+    for extra_check in sorted(extra_checks):
+        print_subheading(f"{extra_check} checks")
+        extra_success = perform_checks(EXTRA_CHECKS_MAP[extra_check])
+        if extra_success:
+            extras_supported.add(extra_check)
+        else:
+            extras_not_supported.add(extra_check)
+
+    return (extras_supported, extras_not_supported)
```

Comment:
Not really important but generally leave off brackets here (I think there's a pylint style warning that guess we have disabled).
```suggestion
    return extras_supported, extras_not_supported
```